app/components/AccountPlan/AccountPlan.tsx

```diff
@@ -12,46 +12,11 @@ import {
 import { LuCheck } from "react-icons/lu"
 import { PayPlanType } from "~/models/portal/sdk"
 
-// TODO_TECHDEBT: Remove all references to PayAsYouGoV0 and FreetierV0` after its live
 type AccountPlanProps = {
```

Comment:
Should we also updated `export enum PayPlanType {`?

Reply:
This enum is autogenerated from the GraphQL schema and in a .gitignored file.

It will be auto updated when deploy the updated UI pointed to the updated PUB GraphQL schema.